libmultilabel/model.py

```diff
@@ -59,40 +60,55 @@ def training_step(self, batch, batch_idx):
         return loss
 
     def validation_step(self, batch, batch_idx):
-        loss, pred_logits = self.shared_step(batch)
-        return {'loss': loss.item(),
-                'pred_scores': torch.sigmoid(pred_logits).detach().cpu().numpy(),
-                'target': batch['label'].detach().cpu().numpy()}
+        self._shared_eval_step(batch, datch_idx)
```

Comment:
```suggestion
        self._shared_eval_step(batch, batch_idx)
```
`datch_idx` is a typo.

Reply:
Good eye. Thanks!